### PR TITLE
Use morgan

### PR DESCRIPTION
**Pull request by djmitche**
As @jhford suggested -- this will log each request on the webhead that performs it, as well as the time spent.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -51,7 +52,7 @@ let load = Loader({
 
         },
         sentryDSN: async (project) => {
-          let key = await this.sentryManager.projectDSN(project);
+          let key = await sentryManager.projectDSN(project);
```
Comment (imbstack) [inline src/main.js]: I thought we made this change yesterday?

**Comment by imbstack** [comment]
Also where are the tests results O_o

File: src/main.js (hunk 1 above)
Comment (djmitche) [reply src/main.js]: Oh no, apparently I didn't land that!  I pushed to origin/master :(

**Comment by djmitche** [comment]
Re test results: EALREADYHAVE99PROBLEMS

**Comment by djmitche** [comment]
I made a followup to put the app.use(morgan..) before the app.use('/v1', ..).

**Pull request merged**